README.md

```diff
@@ -15,6 +15,14 @@ As its name suggests, Thrift is designed to be lightweight and efficient. While
 Before getting started, make sure you have a proper Zephyr development
 environment by following the official
 [Zephyr Getting Started Guide](https://docs.zephyrproject.org/latest/getting_started/index.html).
+When taking the 5th step in [Get Zephyr and install Python dependencies](https://docs.zephyrproject.org/latest/develop/getting_started/index.html#get-zephyr-and-install-python-dependencies), copy [99-thrift.yaml](submanifests/99-thrift.yaml)
```

Comment:
Let's explicitly say after following the GSG (see comments in issue). That makes this more of an additive process change which is better.

After the GSG, we should add the submanifest file and run west update again.

I'm also thinking that instead of the long revision for .upstream, we can simply use 'zephyr' to refer to the branch.